update-bcd.ts

```diff
@@ -299,8 +299,9 @@ export const update = (
       continue;
     }
 
+    const support = entry.__compat.support;
     // Stringified then parsed to deep clone the support statements
-    const originalSupport = JSON.parse(JSON.stringify(entry.__compat.support));
+    const originalSupport = JSON.parse(JSON.stringify(support));
```

Comment:
If you like, feel free to define `const clone = (value) => JSON.parse(JSON.stringify(value))` like in https://github.com/foolip/mdn-bcd-collector/pull/2317 and use it here/elsewhere.